specification/provisioningservices/resource-manager/Microsoft.Devices/2017-08-21-preview/iotdps.json

```diff
@@ -0,0 +1,1600 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "2017-08-21-preview",
+		"title": "iotDpsClient",
+		"description": "API for using the IoT Device Provisioning Service Features",
+		"x-ms-code-generation-settings": {
+			"header": "MICROSOFT_MIT_NO_VERSION"
+		}
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"consumes": [ "application/json" ],
+	"produces": [ "application/json" ],
+	"paths": {
```

Comment:
Is there no PUT , PATCH or List API for certificates

Reply:
We fixed it already, we need to update swagger spec. Karan, please update swagger.